doc/subsystems/disk/sdhc.rst

```diff
@@ -0,0 +1,42 @@
+SDHC disks
+##########
+
+Zephyr includes support for connecting a SDHC card via the SPI bus.
+When used with ELMFS, this allows reading and writing FAT based
+filesystems.
+
+The system has been tested with cards from Samsung, SanDisk, and 4V
+with sizes from 2 GiB to 32 GiB in single partition mode.
+
+MMC and SDSC (<= 2 GiB) cards are not supported and will be ignored.
+
+The system does not support inserting or removing cards, so the card
```

Comment:
maybe add:

    The system does not support inserting or removing cards while the system is running.
    The cards must be present at boot and must not be removed.

Reply:
Done